utils/hare-bootstrap

```diff
@@ -241,11 +241,17 @@ EOF
 
     while read node _; do
         if is_localhost $node; then
-            echo $node > $conf_dir/node-name
+            echo $node > $conf_dir/node-name || {
+                echo "Permission Denied at $node for $conf_dir/node-name"
+                exit 1
+            }
         else
             # Redirect stdin to /dev/null (`-n`) to prevent
             # accidental stealing of `get_all_nodes` output.
-	    ssh -n $node "echo $node > $conf_dir/node-name"
+	    ssh -n $node "echo $node > $conf_dir/node-name" || {
+                echo "Permission Denied at $node for $conf_dir/node-name"
```

Comment:
It is not necessary always must be the "permission denied" error. In case there is some other error, this will cause confusion.

Reply:
Hi @andriytk Yeah I'll edit as suggested. Thanks.